src/state/predictions/index.ts

```diff
@@ -184,6 +208,18 @@ export const predictionsSlice = createSlice({
       state.isFetchingHistory = false
       state.isHistoryPaneOpen = true
       state.history[account] = bets
+
+      // Save any fetched bets in the "bets" namespace
+      const betData = bets.reduce((accum, bet) => {
+        return {
+          ...accum,
+          [bet.round.id]: bet,
+        }
+      }, {})
+
+      state.bets = merge({}, state.bets, {
+        [account]: betData,
+      })
```

Comment:
Every time we fetch the bets to display in the pane we can save the results in the `bets` object.